docs/development/Contributing.md

```diff
@@ -0,0 +1,74 @@
+# You can help bring INAV forward! 
+
+INAV is a big project and fully community driven. This is why we need you, to make INAV better with every release! 
+There are over 100 new features and fixes ready to go into INAV, contributed by a lot of developers. To ensure that 
+all quality standards are met, all these great additions need to be checked, tested and potentially fixed before we 
+can add them to the code. This is why we always need your help, to keep the standards high and our aircraft safe! 
+If you are a bit tech-savvy and curious to try new things in the hobby you are invited to join us and help with the development.
+Also, if you are good at writing in English, you are welcome to help with the documentation.
+
+## How are changes made in INAV? (Testing and reviewing pull requests)
+Most changes done to INAV are done by what’s called a “pull request”, or “PR”. As of this writing, there are currently
+98 pull requests, 98 new things waiting to be added to INAV itself, and 34 for Configurator. Each is waiting for someone
+to test them, or update the related documentation, or just look over the code. If you are curious about whats coming,
+then have a look at the [INAV Pull Requests](https://github.com/iNavFlight/inav/pulls)
+
+And the ones for [Configurator](https://github.com/iNavFlight/inav-configurator/pulls)
+
+Some of the ones ready to be tested are shown here:
+[Testing Required](https://github.com/iNavFlight/inav/pulls?q=is%3Aopen+is%3Apr+label%3A"Testing+Required")
+
+Some are just documentation updates and we’d like someone to review to see if you think it reads clearly and is accurate, such as these:
+[Documentation
+PRs](https://github.com/iNavFlight/inav/pulls?q=is%3Aopen+is%3Apr+label%3A"Review+needed"+label%3ADocumentation)
+
+
+## Documentation Updates
+Another important part of software development is the documentation. Something that INAV struggled with in the past and we want to get better at. Check the new features, read the description from the developer, and make sure a proper documentation of any new feature is provided. Not enough information in the PR? Feel free to make the developers aware of it. You see potential in better guides or descriptions? You are welcome to add or change things, to make them more accessible for newcomers. It is often better for someone BESIDES the developer to write or at least review the documentation for a feature because we want it to be clear and understandable to someone who doesn't already fully understand it.
```

Comment:
```suggestion
Another important part of software development is the documentation. This is something that INAV has struggled with in the past, and we want to get better at it. Check the new features, read the description from the developer, and make sure proper documentation for any new feature is provided. Not enough information in the PR? Feel free to make the developers aware of it. You see potential in better guides or descriptions? You are welcome to add or change things, to make them more accessible for newcomers. It is often better for someone BESIDES the developer to write or at least review the documentation for a feature because we want it to be clear and understandable to someone who doesn't already fully understand it.
```